ReactiveCocoaTests/Swift/FlattenSpec.swift

```diff
@@ -70,6 +70,622 @@ class FlattenSpec: QuickSpec {
 			describeSignalFlattenDisposal(.Merge, name: "merge")
 			describeSignalFlattenDisposal(.Concat, name: "concat")
 		}
+		
+		describe("Signal.flatten()") {
```

Comment:
I think dividing `func describeSignalFlattenDisposal`/`context("Signal")` and `func describeSignalProducerFlattenDisposal`/`context("SignalProducer")` makes the spec unreadable. Moving the `describe`s after `context("SignalProducer")` might be fine.